packages/node/src/channels.ts

```diff
@@ -285,12 +285,34 @@ export class Channels {
           const appInstanceId = await this.store.getAppInstanceIDFromAppInstanceIdentityHash(
             appInstance.identityHash
           );
-          return await this.store.getAppInstanceInfo(appInstanceId);
+          return this.store.getAppInstanceInfo(appInstanceId);
         }
       )
     );
   }
 
+  async getCommitmentForAppInstanceHash(
+    appInstanceHash: string,
+    protocol: string
+  ): Promise<string> {
+    return this.store.getCommitmentForAppInstanceHash(
+      appInstanceHash,
+      protocol
+    );
+  }
+
+  async setCommitmentForAppInstanceHash(
+    appInstanceHash: string,
+    protocol: string,
+    commitment: string
+  ) {
+    return this.store.setCommitmentForAppInstanceHash(
+      appInstanceHash,
+      protocol,
+      commitment
+    );
+  }
+
   getNonFreeBalanceAppInstancesJSON(stateChannel: StateChannel): AppInstance[] {
     return [...stateChannel.appInstances.values()].filter(
```

Comment:
```suggestion
    return stateChannel.appInstances.values().filter(
```

Reply:
Actually furthermore, the `Map` class has native filter ability. Why are we converting to JSON?

Reply:
Don't know. @cf19drofxots